users/romus/romus.c

```diff
@@ -0,0 +1,633 @@
+#include "romus.h"
+
+/*---------------*\
+|*-----MOUSE-----*|
+\*---------------*/
+#ifdef MOUSEKEY_ENABLE
+#include "mousekey.h"
+#endif
+
+/*-------------*\
+|*-----RGB-----*|
+\*-------------*/
+#ifdef RGBLIGHT_ENABLE
+#include "rgblight.h"
+#endif
+
+/*-------------*\
+|*---UNICODE---*|
+\*-------------*/
+#ifdef UNICODE_ENABLE
+#endif
+
+/*-----------------*\
+|*-----SECRETS-----*|
+\*-----------------*/
+// Enabled by adding a non-tracked secrets.h to this dir.
+#if (__has_include("secrets.h"))
```

Comment:
Just a heads up, this can cause compiler errors, in some cases, becasue of how it works.

I've updated the "secret" stuff so that it won't cause this to happen: 
https://github.com/qmk/qmk_firmware/tree/master/users/drashna#secret-macros